wordpress-mobile/WordPress-iOS
# Logged Status Helper

### Details:
- Implements **AccountHelper**
- Nuked a lot of dulpicated code
- Since i was there... added few Groups in the `Utility` Project-Folder

Closes #3409

### To test:
- Verify that the Unit Tests run as expected
- Verify that i didn't (accidentaly) invert the logic anywhere!
- Take the app for a quick ride, with no default Dotcom account, and verify the Reader / Notifications work as expected.

Eric, no more dupe code for this one!! May i bug you (yet again)?
Needs review: @aerych 

Thanks!!!
